# HG changeset patch
# User Ben Abbott <bpabbott@mac.com>
# Date 1287401556 -28800
#      Mon Oct 18 19:32:36 2010 +0800
# Node ID 811b51cdddce443d0c54922f752b2b95bb34ed00
# Parent  36c18286a61bce14104cb60c3786ca739eba129b
print.m: Add examples to the docstring.

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2010-10-18  Ben Abbott <bpabbott@mac.com>
+
+	* plot/print.m: Add examples to the docstring.
+
 2010-10-18  John W. Eaton  <jwe@octave.org>
 
 	* plot/private/__errplot__.m: Remove debugging statement from
 	previous change.
 
 2010-10-17  John W. Eaton  <jwe@octave.org>
 
 	* miscellaneous/edit.m: Use version 3 in GPL notice.
diff --git a/scripts/plot/print.m b/scripts/plot/print.m
--- a/scripts/plot/print.m
+++ b/scripts/plot/print.m
@@ -224,16 +224,36 @@
 ## @item -F@var{fontname}
 ## @itemx -F@var{fontname}:@var{size}
 ## @itemx -F:@var{size}
 ##   Associates all text with the @var{fontname} and/or @var{fontsize}.
 ## @var{fontname} is ignored for some devices; dxf, fig, hpgl, etc.
 ## @end table
 ##
 ## The filename and options can be given in any order.
+##
+## Example: Print to a file, using the svg device.
+##
+## @example
+## figure (1)
+## clf ()
+## surf (peaks)
+## print -dsvg figure1.svg
+## @end example
+##
+## Example: Print to an HP Deskjet 550C.
+##
+## @example
+## figure (1)
+## clf ()
+## surf (peaks)
+## print -dcdj550
+## @end example
+##
+## @seealso{figure, orient}
 ## @end deftypefn
 
 function print (varargin)
 
   opts = __print_parse_opts__ (varargin{:});
 
   opts.pstoedit_cmd = @pstoedit;
   opts.fig2dev_cmd = @fig2dev;
